### ISSUE TITLE
Potential Bug in Skia Renderer when rendering LabelStyle

### ISSUE THREAD

**Issue by PChristoph** [pauldendulk/Mapsui#262]
Dear pauldendulk,

after switching from WPF renderer to skia I noticed a difference while rendering LabelStyles. I have a Feature with a LineString as Geometry. I define a List of Styles and assign it. One of it is a LabelStyle like this:

```
LabelStyle {
                    Text = String.Empty,
                    BackColor = new Brush(Color.Black),
                    ForeColor = Color.White,
                    LabelMethod = f => { return SomehowCreateLabelText(f); }
                };
```

When using the WPF renderer everything works fine. However when switching to Skia the Label won't appear at all.

By the way I had some issues when not setting Text explicitly to String.Empty because I got a null reference Exception from Mapsui. However this was some months ago and with the wpf renderer. Removing the line Text=String.Empty didn't throw an Exception now but didn't fix the issue of this topic.

I have no doubt you will be able to fix that quickly as always. Thank you for your time!

Best regards
Christoph

**Comment by pauldendulk** [comment]
There is nothing special going on with your label style definition. Perhaps it is related to null data as you mention. Setting the Text property has no effect since te LabelMethod will determine the Text if set. 

I released a beta that checks text for null, maybe you can try it:
https://www.nuget.org/packages/Mapsui/1.2.3-beta2

**Comment by pauldendulk** [comment]
This could be related to an issue with polygon/line labels, or an issue with null values of labels, which both have been fixed. I assume this fixed until another report comes in for 1.3.1-beta3 or later.
